package.json

```diff
@@ -19,20 +19,6 @@
   "author": "azu",
   "sideEffects": false,
   "type": "module",
-  "exports": {
-    ".": {
-      "import": {
-        "types": "./module/sentence-splitter.d.ts",
-        "default": "./module/sentence-splitter.js"
-      },
-      "require": {
-        "types": "./lib/sentence-splitter.d.ts",
-        "default": "./lib/sentence-splitter.js"
-      },
-      "default": "./lib/sentence-splitter.js"
-    },
-    "./package.json": "./package.json"
-  },
```

Comment:
What problems have you encountered with `exports`?

Reply:
I've managed to make it work. Just in my application's _tsconfig.json_ I had to change _moduleResolution_ from `node` to `bundler`.

Reply:
Thanks, I see.

TypeScript requires `nodenext`/`node16` or `bundler`as `moduleResolution` for resolving `exports` fields.
`node` does not resolve `exports` fields.

- https://www.typescriptlang.org/tsconfig#moduleResolution
- [Node Package Imports are supported only when moduleResolution is set to &quot;nodenext&quot; or &quot;node16&quot;.](https://stackoverflow.com/questions/58990498/package-json-exports-field-not-working-with-typescript)
- [Support package.json `exports` outside of Node · Issue #50794 · microsoft/TypeScript](https://github.com/microsoft/TypeScript/issues/50794)